InfrastructureManager/agents/vsphere_agent.py

```diff
@@ -0,0 +1,111 @@
+"""
+[TESTING]
+Agent class for vSphere.
+The agent need pysphere 0.1.8 you can install from pip or from esayinstall tools
+
+You need setup your vCenter:
+    you need create a AppScale template with VMWare tools installed and to be
+    powered on.
+      you need create a head node in vSphere
+"""
+
+from agents.base_agent import BaseAgent, AgentConfigurationException, AgentRuntimeException
+import datetime
+from pysphere import VIServer
+import time
+from utils import utils
+
+__author__ = 'Marco Maria Di Nicola'
+__email__ = 'marticus@hotmail.it'
+__email2__ = 'dinicola.1210022@studenti.uniroma1.it'
+
+PARAM_USERNAME = 'vusername'
+PARAM_PASSWORD = 'vpassword'
+PARAM_HOST = 'vhost'
+PARAM_RESOURCE_POOL = 'vresource_pool'       #name of Resouce Pool in vSphere (reservate for AppScale) some like appscale-pool
```

Comment:
Run "pylint --indent-string '  ' <filename>" on all python files. It'll complain about this line being too long.